src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+import { CHECK_BOOK_CATEGORY } from '../constants';
```

Comment:
- [OPTIONAL] Same here, the value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS` Kindly make this 👍🏻

Reply:
I followed this in `constants.js`. Can you have a look at here?
https://github.com/PrangonGhose/bookstore-with-react/blob/145776b7923ccf3e2ec09973c2d946539edec2c4/src/redux/constants.js#L3

Reply:
Hey @PrangonGhose, Sorry for the late reply. Kindly submit for the next reviewer to approve this 👍🏻